code/modules/admin/player_panel.dm

```diff
@@ -231,10 +231,10 @@
 			if(isliving(M))
 
 				if(iscarbon(M)) //Carbon stuff
-					if(ismonkey(M))
-						M_job = "Monkey"
-					else if(ishuman(M))
+					if(ishuman(M) && M.job)
 						M_job = M.job
+					else if(ismonkey(M))
+						M_job = "Monkey"
```

Comment:
Monkeys are now a species datum, while the atom path is `carbon/human`, so this `else` conditional will never pass.

Reply:
brb fucking dying